dcp_prototype/backend/wrangling/migrations/utils/migration_utils.py

```diff
@@ -91,23 +92,22 @@ class DatasetMetadata(object):
 
     def __init__(self):
         self.artifact = MetadataArtifact()
+        self.project = MetadataProject()
         self.entity_data = {}
         self.missing = {}
         self.lock = threading.Lock()
 
     def validate(self, data):
         """Validate the data with the current artifact schema"""
         try:
-            import dcp_prototype
-
             schema_file = pkg_resources.resource_filename(dcp_prototype.__name__, "backend/v0.0.0.json")
             schema_data = open(schema_file).read()
             schema = json.loads(schema_data)
         except FileNotFoundError:
             # FIXME, the schema is not yet committed to main.
-            # validate should fail if the data cannot be validated.
-            print(f"Warning, the schema could not be validates: {schema_file}")
-            return True
+            # Validate should fail if the data cannot be validated.
+            print(f"Warning, the schema could not be validated: {schema_file}")
+            return False
```

Comment:
I agree this is the behavior we want.  But since the schema is not merged into master yet, this change will cause the create_artifact.py script to fail.  You may want to wait to merge until the schema is in place.

Reply:
Oops; sorry, I totally forgot that the schema hadn't been pushed in yet. I reverted this change since the primary motivation for this PR was to remove old code.